lightningd/plugin.c

```diff
@@ -116,7 +117,7 @@ struct plugin_opt {
 };
 
 struct plugins *plugins_new(const tal_t *ctx, struct log_book *log_book,
-			    struct jsonrpc *rpc)
+			    struct jsonrpc *rpc, struct lightningd *ld)
```

Comment:
Maybe should remove log_book and rpc here, and replace by ld->log_book and log->rpc in follow.